components.go

```diff
@@ -603,7 +603,7 @@ func ParseComponent(cs *CalendarStream, startLine *BaseProperty) (ComponentBase,
 		}
 		line := ParseProperty(*l)
 		if line == nil {
-			return cb, errors.New("Error parsing line")
+			continue
```

Comment:
This also. It feels like a large change?

Reply:
Yes, you right. You lib does not support custom properties(https://datatracker.ietf.org/doc/html/rfc5545#section-3.8.8.2). I will try to support this